src/clients/storage/CassandraTableStorageManager.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const asyncEachLimit = require('async/eachLimit');
+
+/** http://docs.datastax.com/en/developer/nodejs-driver/3.2/features/batch/
+ * TODO: ASYNC_BATCH_LIMIT will need to be tuned.
+ * It is best to keep the batch size small (in the order of 10s).
+ * If the batch size is greater than 5k, then the server will issue a warning.
+ */
+const ASYNC_BATCH_LIMIT = 10;
```

Comment:
Not a deal-breaker for me, but I figured I'd ask: did we decide against using an environment variable for this and defaulting to 10 if it isn't present?